diff -Nru '--exclude=.git' MaxScale-21.06.18.org/cmake/BuildMongoCxxDriver.cmake MaxScale-21.06.18/cmake/BuildMongoCxxDriver.cmake
--- MaxScale-21.06.18.org/cmake/BuildMongoCxxDriver.cmake	2025-03-05 03:19:08.598979662 +0700
+++ MaxScale-21.06.18/cmake/BuildMongoCxxDriver.cmake	2025-03-05 04:18:09.724049941 +0700
@@ -14,17 +14,16 @@
 
 message(STATUS "Using mongo-cxx-driver version ${MONGO_CXX_DRIVER_VERSION}")
 
-set(MONGO_CXX_DRIVER_URL "https://github.com/mongodb/mongo-cxx-driver/releases/download/r${MONGO_CXX_DRIVER_VERSION}/mongo-cxx-driver-r${MONGO_CXX_DRIVER_VERSION}.tar.gz"
-  CACHE STRING "mongo-cxx-driver sources")
+# set(MONGO_CXX_DRIVER_URL "https://github.com/mongodb/mongo-cxx-driver/releases/download/r${MONGO_CXX_DRIVER_VERSION}/mongo-cxx-driver-r${MONGO_CXX_DRIVER_VERSION}.tar.gz"
+#   CACHE STRING "mongo-cxx-driver sources")
 
-set(MONGO_CXX_DRIVER "${CMAKE_BINARY_DIR}/mongo-cxx-driver")
+set(MONGO_CXX_DRIVER "${CMAKE_SOURCE_DIR}/mongo-cxx-driver")
 
 set(MONGO_CXX_DRIVER_SOURCE  "${MONGO_CXX_DRIVER}/src")
 set(MONGO_CXX_DRIVER_BINARY  "${MONGO_CXX_DRIVER}/build")
 set(MONGO_CXX_DRIVER_INSTALL "${MONGO_CXX_DRIVER}/install")
 
 ExternalProject_Add(mongo-cxx-driver
-  URL ${MONGO_CXX_DRIVER_URL}
   SOURCE_DIR ${MONGO_CXX_DRIVER_SOURCE}
   CMAKE_ARGS -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} -DCMAKE_INSTALL_PREFIX=${MONGO_CXX_DRIVER_INSTALL} -DCMAKE_PREFIX_PATH=${MONGO_C_DRIVER_INSTALL} -DCMAKE_C_FLAGS=-fPIC  -DCMAKE_CXX_FLAGS=-fPIC -DBUILD_SHARED_LIBS=OFF -DBUILD_STATIC_LIBS=ON -DBUILD_SHARED_AND_STATIC_LIBS=OFF
   BINARY_DIR ${MONGO_CXX_DRIVER_BINARY}
